### ISSUE TITLE
Map GENO to HPO inheritance/dominance

### ISSUE THREAD

**Issue by mbrush** [monarch-initiative/GENO-ontology#6]
**Transferred from:** [NIF-11647](https://support.crbs.ucsd.edu/browse/NIF-11647)
**Original Reporter:** Nicole Washington

---

i notice that HPO has some dominance/inheritance terms. these should be reconciled some way with GENO.

for example:
http://www.human-phenotype-ontology.org/hpoweb/showterm?id=HP:0000006

**Comment by pnrobinson** [comment]
Please note that the HPO terms are defined in a way that makes sense for humans. Under the assumption that GENO will not want to be guilty of rampant speciesism please consider for instance that X/Y are not universal:
http://en.wikipedia.org/wiki/ZW_sex-determination_system
One solution would be to somehow define the HPO terms as descendents of some UBER inheritance terms.
Also, whenever it might be useful I would take a look at the human mutation/genotype terms, we really should ensure that it is compatible to HGVS stuff, which occasionally is a little opaque.

cheers Peter

Dr. med. Peter N. Robinson, MSc.
Professor of Medical Genomics
Professor in the Bioinformatics Division of the Department of Mathematics and Computer Science of the Freie Universität Berlin
Institut für Medizinische Genetik und Humangenetik
Charité - Universitätsmedizin Berlin
Augustenburger Platz 1
13353 Berlin
Germany
+4930 450566006
Mobile: 0160 93769872
peter.robinson@charite.de
http://compbio.charite.de
http://www.human-phenotype-ontology.org
Introduction to Bio-Ontologies: http://www.crcpress.com/product/isbn/9781439836651
I have learned from my mistakes, and I am sure I can repeat them exactly
ORCID ID:http://orcid.org/0000-0002-0736-9199
Scopus Author ID 7403719646
Appointment request: http://doodle.com/pnrobinson

---

Von: mbrush [notifications@github.com]
Gesendet: Freitag, 16. Januar 2015 02:01
An: monarch-initiative/GENO-ontology
Betreff: [GENO-ontology] Map GENO to HPO inheritance/dominance (#6)

Transferred from: NIF-11647https://support.crbs.ucsd.edu/browse/NIF-11647
Original Reporter: Nicole Washington

---

i notice that HPO has some dominance/inheritance terms. these should be reconciled some way with GENO.

for example:
http://www.human-phenotype-ontology.org/hpoweb/showterm?id=HP:0000006

**Comment by mellybelly** [comment]
I agree, we should get these into GENO in a generic way. The HPO terms can be either subclasses or HPO-specific labels and end-user descriptions if needed. We could also just start only with organisms that in fact do have XY sex determination mechanisms. 

See also related terms in PATO having to do with sexual development and dimorphism.
